account_asset_management/models/account_move.py

```diff
@@ -59,8 +59,11 @@ def write(self, vals):
         return super().write(vals)
 
     def post(self):
+        AccountAsset = self.env["account.asset"]
         for move in self:
-            for aml in move.line_ids.filtered("asset_profile_id"):
```

Comment:
Why removing this valid code?

Reply:
Because creating a new list to loop again on it is not needed

Reply:
But it's not a new "list". It's clearer as it was.

Reply:
https://github.com/odoo/odoo/blob/13.0/odoo/models.py#L5246

Reply:
Yes, but now in v13 cache is shared, and you get no penalty of it.